composed_api.rst

```diff
@@ -316,34 +305,144 @@ Example
 
 ::
 
-    curl -u <username> http://gridte.rsgis.erdc.dren.mil/te_ba/api/v1/aoi/add/?source=grid&name=test&geom=POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10))&subscribe=True
+    curl -u <username> http://gridte.rsgis.erdc.dren.mil/te_ba/api/v2/aoi/add/?source=grid&name=test&geom=POLYGON ((30 10, 40 40, 20 40, 10 20, 30 10))&subscribe=True
 
 ::
 
     {
-        "1592": {
-            "aoi": [
-                {
-                    "fields": {
-                        "clip_geometry": "SRID=4326;POLYGON ((30.0000000000000000 10.0000000000000000, 40.0000000000000000 40.0000000000000000, 20.0000000000000000 40.0000000000000000, 10.0000000000000000 20.0000000000000000, 30.0000000000000000 10.0000000000000000))", 
-                        "created_at": "2015-11-13T12:58:28.040", 
-                        "is_active": true, 
-                        "name": "test", 
-                        "notes": "", 
-                        "source": "api", 
-                        "user": 102
-                    }, 
-                    "model": "export.aoi", 
-                    "pk": 1592
-                }
-            ], 
-            "export_set": [], 
-            "pointcloud_collects": [], 
-            "raster_collects": []
-        }, 
-        "GRiD API": {
-            "API Version": "v1"
-        }, 
+        "aoi": 
+        {
+          "clip_geometry": "SRID=4326;POLYGON ((30.0000000000000000 10.0000000000000000, 40.0000000000000000 40.0000000000000000,
+          20.0000000000000000 40.0000000000000000, 10.0000000000000000 20.0000000000000000, 30.0000000000000000 10.0000000000000000))", 
+          "created_at": "2015-11-13T12:58:28.040", 
+          "is_active": true, 
+          "name": "test", 
+          "notes": "", 
+          "source": "api", 
+          "user": 102,
+          "pk": 1592
+        },
+        "export_set": [], 
+        "pointcloud_intersects": [], 
+        "raster_intersects": [],
+        "API Version": "v2",
+        "success": true
+    }
+
+Edit AOI
+~~~~~~~~
+
+Update an AOIs name, notes, or geometry. In order to change an AOI's
+geometry, it must contain 0 generated exports.
+
+Endpoint
+^^^^^^^^
+
+::
+
+    GET <instance_url>/<instance_root>_ba/api/v2/aoi/edit/<pk>
```

Comment:
There is an argument here that we should not be using verbs as endpoints, rather hit the `/aoi` endpoint with a `PUT` for an edit, `GET` for a read, `POST` for an add, and `DELETE` for well, deleting. This will impact several endpoints.